### ISSUE TITLE
cicero verify does not report errors properly

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/cicero#729]
# Bug Report 🐛

Cicero verify does not report errors properly.

## Steps to Reproduce

In `./packages/cicero-core`:

```
zsh-5.8$ ../cicero-cli/index.js verify --template test/data/verifying-template-signature/helloworldstateSigned 
cicero verify

verify the template signatures of the template author/developer

Options:
      --version   Show version number                                  [boolean]
  -v, --verbose                                                 [default: false]
      --help      Show help                                            [boolean]
      --template  path to the template                                  [string]
      --warnings  print warnings                      [boolean] [default: false]

TypeError: Cannot read property 'error' of undefined
```

**Comment by jeromesimeon** [comment]
It also silently reports success:

```
zsh-5.8$ ../cicero-cli/index.js verify --template helloworldstate@0.15.0.cta 
zsh-5.8$
```

**Comment by jeromesimeon** [comment]
Only affected `0.23.0-alpha.1` (dependency issue in the CLI). Fixed in #730
